build.gradle.kts

```diff
@@ -57,21 +63,20 @@ dependencies {
     // Build environment
     implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.2")
-    detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.20.0-RC1")
+    detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.20.0")
     testImplementation("org.junit.jupiter:junit-jupiter:5.8.2")
     implementation("com.cognifide.gradle:common-plugin:1.1.8")
 
     // External dependencies
     implementation("org.jsoup:jsoup:1.14.3")
     implementation("org.buildobjects:jproc:2.8.0")
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.2")
     implementation("org.apache.commons:commons-lang3:3.12.0")
     implementation("commons-io:commons-io:2.11.0")
-    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.13.1")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.13.3")
```

Comment:
that some day broke AEM instances connectivity. you should try running `instanceUp` task after upgrading it to be sure if it still works

Reply:
My bad, I saw vulnerability here, so I upgrade version.

Reply:
it's ok, but you need to retest GAP instanceAwait/instanceUp task at least once

Reply:
InstanceAwait returns:

```
> Task :env:local:aem:instanceAwait
Caching disabled for task ':env:local:aem:instanceAwait' because:
  Caching has not been enabled for the task
Task ':env:local:aem:instanceAwait' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
Successfully read product version '2022.7.8005.20220711T194049Z' of LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Successfully read product version '2022.7.8005.20220711T194049Z' of LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Successfully read status properties at path '/system/console/status-System Properties.txt' on LocalInstance(name='local-author', httpUrl='http://localhost:4502').
Successfully read status properties at path '/system/console/status-System Properties.txt' on LocalInstance(name='local-publish', httpUrl='http://localhost:4503').
Awaiting instance(s) up: local-author, local-publish

Checking OSGi bundles on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking OSGi events on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking OSGi installer busyness on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking OSGi installer pause on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking OSGi components on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking OSGi bundles on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking OSGi events on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking OSGi installer busyness on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking OSGi installer pause on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking OSGi components on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Awaiting state to be changed on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Awaiting state to be changed on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking OSGi bundles on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking OSGi events on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking OSGi installer busyness on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking OSGi installer pause on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking OSGi components on LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking OSGi bundles on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking OSGi events on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking OSGi installer busyness on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking OSGi installer pause on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking OSGi components on LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking done for LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking ended for LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking done for LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Checking ended for LocalInstance(name='local-author', httpUrl='http://localhost:4502')
:env:local:aem:instanceAwait (Thread[Execution worker for ':' Thread 3,5,main]) completed. Took 5.709 secs.

BUILD SUCCESSFUL in 8s
13 actionable tasks: 1 executed, 12 up-to-date
Watched directory hierarchies: [C:\projects\gat]
```

InstanceUp returns:

```
> Task :env:local:aem:instanceUp
Caching disabled for task ':env:local:aem:instanceUp' because:
  Caching has not been enabled for the task
Task ':env:local:aem:instanceUp' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
Successfully read product version '2022.7.8005.20220711T194049Z' of LocalInstance(name='local-author', httpUrl='http://localhost:4502')
Successfully read product version '2022.7.8005.20220711T194049Z' of LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Successfully read status properties at path '/system/console/status-System Properties.txt' on LocalInstance(name='local-author', httpUrl='http://localhost:4502').
Successfully read status properties at path '/system/console/status-System Properties.txt' on LocalInstance(name='local-publish', httpUrl='http://localhost:4503').
No instances to turn on.
Awaiting instance(s) up: local-author, local-publish

Checking done for LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Checking ended for LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Customizing: LocalInstance(name='local-publish', httpUrl='http://localhost:4503')
Customizing: LocalInstance(name='local-author', httpUrl='http://localhost:4502')
:env:local:aem:instanceUp (Thread[Execution worker for ':',5,main]) completed. Took 2.323 secs.

BUILD SUCCESSFUL in 4s
14 actionable tasks: 2 executed, 12 up-to-date
Watched directory hierarchies: [C:\projects\gat]
```

Can we assume version upgrade didn't break anything?

Reply:
No. No instances to turn on means that you didn't check it properly